modules/nickserv/verify.c

```diff
@@ -190,6 +190,9 @@ ns_cmd_fverify(struct sourceinfo *si, int parc, char *parv[])
 		metadata_delete(mu, "private:verify:emailchg:newemail");
 		metadata_delete(mu, "private:verify:emailchg:timestamp");
 
+		// tell the user they have been verified
+		myuser_notice(nicksvs.nick, mu, _("\2%s\2 has now been verified."), entity(mu)->name);
```

Comment:
not 100% on the wording here. might not make as much sense when it's not directly after you've personally executed a verification command

Reply:
agreed, i'd still send a message but indicate something like "\2%s\2 has been verified by network staff" or similar

in the future, we might want this to allow off-IRC email verifications (e.g. via xmlrpc or some other api) as that's something that people have expressed interest in, but imo that's something to consider once we add an API for that (calling fverify via atheme.command is imo not a great way to do it)